glanzb/cards4-template.js

```diff
@@ -0,0 +1,142 @@
+var makeCard = // receive factory with external name `makeCard`
```

Comment:
Overall, looks good! A couple of problem spots to work through, but good work throughout.